examples/tutorials/mocking.md

```diff
@@ -0,0 +1,594 @@
+---
+title: "Testing in isolation with mocks"
+description: "Master the art of mocking in your unit tests in with this comprehensive guide. Learn how spies, stubs, and fake time, and other Deno helpers let you to isolate your code during testing, making your unit tests more reliable and efficient."
```

Comment:
While this is a good intro / teaser, I think this is too long for a meta description. Sadly these need to be pretty short in order to perform well. (and work in our OG images)

It gives me the ick a little, but tools like this can sometime help with crafting titles and meta descriptions: https://rankmath.com/tools/meta-tag-analyzer/

FWIW I muddle around in it for a few minutes and came back with this:

```suggestion
description: "Master the art of mocking in your unit tests. Learn how spies, stubs, fake time, and other Deno tools let you improve your code and confidence"
```